app/mailers/concerns/autoresponder.rb

```diff
@@ -0,0 +1,9 @@
+module Autoresponder
+  include ActiveSupport::Concern
+
+  def autorespond(parsed_email)
+    mail(from: noreply_address,
+         to: parsed_email.from.to_s,
+         subject: 'This mailbox is not monitored')
+  end
```

Comment:
Does this actually do anything? I saw it in the old app, but couldn't understand its mechanism of operation.